runbotics/runbotics-orchestrator-ui/src/src-app/views/credentials/Credential/EditCredential/EditCredential.utils.ts

```diff
@@ -34,3 +34,11 @@ export const getInitialCredentialData = (collectionId: string): CreateCredential
     collectionId: collectionId ? collectionId : '',
     templateId: ''
 });
+
+export const isCreatedNow = (dateString: string): boolean => {
+    const inputDate = new Date(dateString);
+    const now = new Date();
+    const timeDifference = now.getTime() - inputDate.getTime();
+
+    return timeDifference <= 60 * 1000 && timeDifference >= 0;
```

Comment:
I would assign `60 * 1000` to named variable, and maybe I would also name this condition `timeDifference <= 60 * 1000 && timeDifference >= 0`, it can be hard to understand later